com.unity.ml-agents/Runtime/Utilities.cs

```diff
@@ -75,83 +74,6 @@ public static int[] CumSum(int[] input)
-        /// List whose elements will be shifted
-        /// </param>
-        /// <param name="shiftAmount">
-        /// Amount to shift the elements to the left by
-        /// </param>
-        /// </summary>
-        public static void ShiftLeft<T>(List<T> list, int shiftAmount)
-        {
-            for (var i = shiftAmount; i < list.Count; i++)
-            {
-                list[i - shiftAmount] = list[i];
-            }
-        }
-
-        /// <summary>
-        /// Replaces target list elements with source list elements starting at specified position
-        /// in target list.
-        /// <param name="dst">
-        /// Target list, where the elements are added to
-        /// </param>
-        /// <param name="src">
-        /// Source array, where the elements are copied from
-        /// </param>
-        /// <param name="start">
-        /// Starting position in target list to copy elements to
-        /// </param>
-        /// </summary>
-        public static void ReplaceRange<T>(List<T> dst, List<T> src, int start)
-        {
-            for (var i = 0; i < src.Count; i++)
-            {
-                dst[i + start] = src[i];
-            }
-        }
-
-        /// <summary>
-        /// Adds elements to list without extra temp allocations (assuming it fits pre-allocated
-        /// capacity of the list). The built-in List/<T/>.AddRange() unfortunately allocates
-        /// a temporary list to add items (even if the original array has sufficient capacity):
-        /// https://stackoverflow.com/questions/2123161/listt-addrange-implementation-suboptimal
-        /// Note: this implementation might be slow with a large source array.
-        /// <param name="dst">
-        /// Target list, where the elements are added to
-        /// </param>
-        /// <param name="src">
-        /// Source array, where the elements are copied from
-        /// </param>
-        /// </summary>
-        // ReSharper disable once ParameterTypeCanBeEnumerable.Global
-        public static void AddRangeNoAlloc<T>(List<T> dst, T[] src)
-        {
-            // ReSharper disable once LoopCanBeConvertedToQuery
-            foreach (var item in src)
-            {
-                dst.Add(item);
-            }
-        }
-
-        /// <summary>
-        /// Calculates the number of uncompressed floats in a list of ISensor
-        /// </summary>
-        public static int GetSensorFloatObservationSize(this List<ISensor> sensors)
-        {
-            int numFloatObservations = 0;
-            for (var i = 0; i < sensors.Count; i++)
-            {
-                if (sensors[i].GetCompressionType() == SensorCompressionType.None)
-                {
-                    numFloatObservations += sensors[i].ObservationSize();
-                }
-            }
-            return numFloatObservations;
-        }
-
 #if DEBUG
         internal static void DebugCheckNanAndInfinity(float value, string valueCategory, string caller)
```

Comment:
```suggestion
        public static void DebugCheckNanAndInfinity(float value, string valueCategory, string caller)
```

Reply:
Made the other methods in the class internal too.